### ISSUE TITLE
Why it need to remove firewalld and NetworkManager packages

### ISSUE THREAD

**Issue by gouyang** [kubevirt/kubevirt-ansible#52]
I'm trying to preparing kubernetes cluster and deploy KubeVirt on my fedora 27 workstation, my system crashed because the playbook removes NetworkManager and all its dependencies. I know it's better to use a server to run the playbooks, but it would be better to be able run it on user's workstation, especially for those who just come to the project.

**Comment by fabiand** [comment]
Did you find out why NM was removed?

**Comment by gouyang** [comment]
The openshift-ansible and kubespray project can also deploy kubernetes cluster without remove these packages. By looking into the the script `kubernetes/roles/prerequisites/tasks/main.yml`, it maybe used for enable the bridge network, but I'm not sure of it.

**Comment by gouyang** [comment]
Remove NetworkManager will remove lots of dependency packages on user's workstation and then the system crash, even not recoverable.

**Comment by dyasny** [comment]
NetworkManager is required by openshift-ansible, has to be installed, enabled and started (seen on 3.7 and 3.9 so far)

**Comment by gouyang** [comment]
> Okay. So if ths happens in the flow wher ekubernetes or openshift are getting deployed, then this is not a bug, because for these deployments it might be necessary.

I was meaning `openshift-ansible` and `kubespray` could deploy kubernetes cluster successfully without remove NM package.

**Comment by dyasny** [comment]
> So it's not necessary to remove NM to deploy kubernetes cluster.

I have tried to deploy openshift-ansible on a set of nodes where NetworkManager was not installed, and it failed with a dependency error. I had to install, start and enable NetworkManager for openshift-ansible to run a successful deployment

**Comment by gouyang** [comment]
`ansible-playbook -i inventory playbooks/cluster/kubernetes/config.yml` need to remove NM as the issue reported.

` ansible-playbook -i inventory -e "openshift_ansible_dir=/root/openshift-ansible openshift_playbook_path=playbooks/byo/config.yml  openshift_ver=3.7" playbooks/cluster/openshift/config.yml` meet below error which need NM installed and enabled.

```
TASK [openshift_node_dnsmasq : fail] *************************************************************************************************************
Thursday 01 March 2018  10:35:08 +0000 (0:00:00.029)       0:05:03.415 ******** 
fatal: [bootp-73-3-160.eng.pek2.redhat.com]: FAILED! => {
    "changed": false
}

MSG:

Currently, NetworkManager must be installed and enabled prior to installation.
```

I think role `kubernetes-prerequisites` need to make some changes to avoid remove NM as it's not necessary.

**Comment by alexxa** [comment]
@gouyang Yes, we know about it. I think to add a documentation note about it. 

That task comes from OpenShift playbooks. And there are several opened issues about it. E.g.: https://github.com/openshift/openshift-ansible/issues/4950

**Comment by alexxa** [comment]
@gouyang @dyasny are you ok to close this issue because of merged #143?

**Comment by gouyang** [comment]
Okay to close the issue
